feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/confirm/ConfirmStakingFragment.kt

```diff
@@ -117,5 +107,13 @@ class ConfirmStakingFragment : BaseFragment<ConfirmStakingViewModel>() {
 
             bondedAmount?.let { confirmStakeAmount.amountInput.setText(it.toString()) }
         }
+
+        viewModel.unstakingTime.observe {
+            confirmStakingUnstakingPeriodLength.text = getString(R.string.staking_hint_unstake_format, it)
+        }
+
+        viewModel.eraHoursLength.observe {
+            confirmStakingEachEraLength.text = getString(R.string.staking_hint_rewards_format, it)
+        }
```

Comment:
We commonly format strings in ViewModel itself (do in background as much work as possible + unification with other parts of project)

Reply:
Done